pom.xml

```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>com.exasol</groupId>
             <artifactId>bucketfs-java</artifactId>
-            <version>2.4.1</version>
+            <version>2.6.0</version>
```

Comment:
I just realized that you introduced a breaking change in the API (ip --> host). I overlooked that in my review. That is actually a version 3.0.0.

Reply:
I was aware of the change being a breaking one but rated a single renamed method not being worth to increase the major version. As the release has been done I am not sure if we should plan to change the version.